helper/schema/schema_test.go

```diff
@@ -3947,6 +3947,33 @@ func TestSchemaMap_Validate(t *testing.T) {
 			Err: false,
 		},
 
+		"Good sub-resource, interpolated value": {
```

Comment:
I see, it's passing because `Err: true,`. 😉 

You'll just need to change `Required` to `Optional` here to pass the test, since there there is no real value here.